deploy-helm/flux-helm-release-crd.yaml

```diff
@@ -31,6 +31,8 @@ spec:
               format: int64
             resetValues:
               type: boolean
+            skipDepUpdate:
```

Comment:
Since this pertains to charts from git only (published charts _always_ have their dependencies, as I understand it), should it not go in the `.spec.chart.git` section?

Reply:
Technically no.  Published charts don't always have their dependencies.  Sometimes they do. Sometimes they don't.  So it's not really appropriate in the git section.

Reply:
I was under the impression that helm won't let you package a chart without its dependencies, implying that any published chart _must_ include its dependencies. The helm operator doesn't attempt to download the dependencies unless the chart came from git.